src/dtype.rs

```diff
@@ -169,7 +169,15 @@ impl DataType {
 }
 
 /// Represents that a type can be an element of `PyArray`.
-pub trait Element: Clone + Send {
+///
+/// A type `T` that implements this trait should be safe when managed in numpy array,
```

Comment:
I think adding the header
```rust
/// # Safety
```
before this section would be good so that [Clippy's `missing_safety_doc`](https://rust-lang.github.io/rust-clippy/master/#missing_safety_doc) is appeased.

Reply:
Thanks! Could you please take a glance at the revised version?

Reply:
Looks good to me.